src/components/backend-ai-data-view.ts

```diff
@@ -377,16 +411,47 @@ export default class BackendAIData extends BackendAIPage {
           @change="${() => this._validateFolderName()}" pattern="^[a-zA-Z0-9\._-]*$"
             required validationMessage="${_t('data.Allowslettersnumbersand-_dot')}" maxLength="64"
             placeholder="${_t('maxLength.64chars')}"></mwc-textfield>
-          <mwc-select class="full-width fixed-position" id="add-folder-host" label="${_t('data.Host')}" fixedMenuPosition>
-            ${this.vhosts.map((item, idx) => html`
-              <mwc-list-item hasMeta value="${item}" ?selected="${item === this.vhost}">
-                <span>${item}</span>
-                <mwc-icon-button slot="meta" icon="info"
-                    @click="${(e) => this._showStorageDescription(e, item)}">
+          <mwc-select
+            class="full-width fixed-position"
+            id="add-folder-host"
+            label="${_t("data.Host")}"
+            fixedMenuPosition
+            @selected=${(e)=> this.selectedVhost = e.target.value}
+          >
+            ${this.vhosts.map((item, idx) => {
+              let percentage = this.storageProxyInfo[item] && this.storageProxyInfo[item]?.percentage;
+              return html` <mwc-list-item
+                hasMeta
+                value="${item}"
+                ?selected="${item === this.vhost}"
+              >
+                <div class="horizontal layout justified center">
+                  <span>${item}</span>
+                  ${true
+                    ? html`
```

Comment:
I think this condition is always true. Is there a reason why you added this?

Reply:
Sorry, I forgot to add a comment. After https://github.com/lablup/backend.ai/pull/1170 is merged,  I will update this part as a condition of visibility.

Reply:
Oh! you already removed it! Thank you